scripts/build_android.sh

```diff
@@ -52,24 +52,35 @@ CMAKE_ARGS=()
 # target architecture and need an exact version match.
 CMAKE_ARGS+=("-DCAFFE2_CUSTOM_PROTOC_EXECUTABLE=$CAFFE2_ROOT/build_host_protoc/bin/protoc")
 
+CMAKE_ARGS+=($@)
+USE_ARM_COMPUTE=false
+for arg in ${CMAKE_ARGS[@]};do
+  if [ $arg == "-DUSE_ARM_COMPUTE=ON" ];then
+    USE_ARM_COMPUTE=true
+  fi
+done
 # Use android-cmake to build Android project from CMake.
 CMAKE_ARGS+=("-DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK/build/cmake/android.toolchain.cmake")
 
 # Don't build artifacts we don't need
-CMAKE_ARGS+=("-DBUILD_TEST=OFF")
+CMAKE_ARGS+=("-DBUILD_TEST=ON")
```

Comment:
I think this is on by default, either way the comment above doesn't make sense anymore

Reply:
Should we turn on the test by default?

Reply:
Better keep it off. When needed, tests can be built with `scripts/build_android.sh -DBUILD_TEST=ON`